apps/crn-frontend/src/shared-research/ResearchOutput.tsx

```diff
@@ -13,6 +13,16 @@ const ResearchOutput: React.FC = () => {
   const { researchOutputId } = useRouteParams(
     sharedResearch({}).researchOutput,
   );
+
+  const publishedNowPath = useRouteMatch({
+    path: `${
+      sharedResearch({}).researchOutput({ researchOutputId }).$
+    }/publishedNow`,
```

Comment:
Shouldn't this route be with the other ones? I also wonder if it should have been a query parameter rather than a different route

Reply:
It is with the other, corrected now.